tfrecords/create_tfrecords_str.py

```diff
@@ -41,45 +43,43 @@ def load_sidewalks_dataset(args):
 
 def normalize_img(image: np.ndarray, label: np.ndarray) -> Tuple[tf.Tensor, tf.Tensor]:
     image = tf.cast(image, tf.float32) / 255.0
-    label -= 1
+    return image, label
```

Comment:
I suggest we remove it from here too so as to delegate the preprocessing logic to `Transform`.